src/app.js

```diff
@@ -3,11 +3,255 @@ import $ from 'jquery';
+//////////////////////////////////////////////////////////////
+
+// Create a new collection of all trips
+const tripList = new TripList();
+
+// Create a template for all trips to append to the page
+const listTemplate = _.template($('#trip-template').html());
+
+// Render all trips
+const renderList = function renderList() {
+  $('#trip-list').html('');
+  tripList.forEach((trip) => {
+    let generatedHtml = listTemplate(trip.attributes);
+    $('#trip-list').append(generatedHtml);
+    $(`#trip${ trip.id }`).click(function() {
+      getTrip(trip.id);
+      // let singleTrip = tripList.findWhere({id: tripID});
+    });
+  });
+  $('th.sort').removeClass('current-sort-field');
+  $(`th.sort.${ tripList.comparator }`).addClass('current-sort-field');
+};
+
+
+////////////////////////////////////////////////////////////////
+//////////////////////// SINGLE TRIP ///////////////////////////
+////////////////////////////////////////////////////////////////
+
+const singleTripTemplate = _.template($('#trip-info-template').html());
+
+// Get one Trip
+const getTrip = function getTrip(tripID) {
+  $('#create-trip-form').hide();
+  $('#trip-info').html('');
+  let singleTrip = tripList.findWhere({id: tripID});
+  singleTrip.fetch({
+    success: (model, response) => {
+      console.log('Model: ' + singleTrip.parse(model));
+      console.log('Response: ' + response);
+      showTrip(model);
+    },
+  });
+};
+
+// Render One Trip
+const showTrip = function showTrip(trip) {
+  let generatedHtml = singleTripTemplate(trip.attributes);
+  $('#trip-info').html('');
+  $('#trip-info').append(generatedHtml);
+  $('#reserve-form').on('submit', (event) => {
+    event.preventDefault();
+    reserveTripHandler();
+  });
+};
+
+//////////////////////////////////////////////////////////////
+//////////////////////// CREATE TRIP /////////////////////////
+//////////////////////////////////////////////////////////////
+
+const newTripHandler = () => {
+  let tripData = {};
+  TRIP_FIELDS.forEach((field) => {
+    const inputElement = $(`#new-trip-form input[name="${ field }"]`);
+    const value = inputElement.val();
+    tripData[field] = value;
+  });
+
+  let newTrip = new Trip(tripData);
+
+  if (!newTrip.isValid()) {
+    displayErrors(newTrip.validationError);
+    return;
+  }
+
+  newTrip.save({}, {
+    success: (model, response) => {
+      tripList.add(model);
+    },
+    error: (model, response) => {
+      console.log(response);
+    },
+  });
+};
+
+/////////////////////////////////////////////////////////////////
+//////////////////////// RESERVE TRIP ///////////////////////////
+/////////////////////////////////////////////////////////////////
+
+const reserveTripHandler = () => {
+  let reservation = {};
+  RESERVE_FIELDS.forEach((field) => {
+    const inputElement = $(`#reserve-form [name="${ field }"]`);
+    const value = inputElement.val();
+
+    reservation[field] = value;
+
+    inputElement.val('');
```

Comment:
I suggest only clearing the input values on a successful save.  Otherwise you're clearing the tripId and there's no way for a user to enter that.